src/displayapp/fonts/README.md

```diff
@@ -6,47 +6,68 @@
-* Click on Convert, and download the file `jetbrains_mono_bold_20.c` and copy it in `src/DisplayApp/Fonts`
-* Add the font .c file path to src/CMakeLists.txt
-* Add an LV_FONT_DECLARE line in src/libs/lv_conf.h
-
-Add new symbols:
-
-* Browse the [cheatsheet](https://fontawesome.com/cheatsheet/free/solid) and find your new symbols
-* For each symbol, add its hex code (0xf641 for the 'Ad' icon, for example) to the *Range* list (Remember to keep this
-  readme updated with newest range list)
-* Convert this hex value into a UTF-8 code
-  using [this site](http://www.ltg.ed.ac.uk/~richard/utf-8.cgi?input=f185&mode=hex)
-* Define the new symbols in `src/displayapp/screens/Symbols.h`:
-
-```
-static constexpr const char* newSymbol = "\xEF\x86\x85";
-```
-
-## Simple method to generate a font
-
-If you want to generate a basic font containing only numbers and letters, you can use the above settings but instead of specifying a range, simply list the characters you need in the Symbols field and leave the range blank. This is the approach used for the PineTimeStyle watchface.
-This works well for fonts which will only be used to display numbers, but will fail if you try to add a colon or other punctuation.
-
-* Open the [LVGL font converter](https://lvgl.io/tools/fontconverter)
-* Name : open_sans_light
-* Size : 150
-* Bpp : 1 bit-per-pixel
-* Do not enable font compression and horizontal subpixel hinting
-* Load the file `open_sans_light.tff` (use the file in this repo to ensure the version matches) and specify the following symbols : `0123456789`
-* Click on Convert, and download the file `open_sans_light.c` and copy it in `src/DisplayApp/Fonts`
-* Add the font .c file path to src/CMakeLists.txt (search for jetbrains to find the appropriate location/format)
-* Add an LV_FONT_DECLARE line in src/libs/lv_conf.h (as above)
-
-#### Navigation font
+Following you'll find all the necessary instructions to generate new and existing fonts using the [LVGL font converter](https://lvgl.io/tools/fontconverter).
+
+If you are adding new glyphs to an existing font have a look at the section regarding the specific font you are modifying.
+If you are introducing a new font have a look [here](#adding-a-new-font).
+
+### Jetbrains Mono Bold 20
+
+Fill the font converter fields as follows:
+
+- **Name:** `jetbrains_mono_bold_20`
+- **Size:** 20
+- **Bpp:** 1 bit-per-pixel
+- Do *not* enable font compression or horizontal subpixel hinting
+- Do *not* enable use of glyph color
+- Load `JetBrainsMono-Bold.tff` (use the file in this repo to ensure the version matches)
+- **Range:** `0x20-0x7f,0x410-0x44f` and add any additional glyphs
+- Include a second font
+- Load `FontAwesome5-Solid+Brands+Regular.woff` (use the file in this repo to ensure the version matches)
```

Comment:
This file exists in a different place, so perhaps there should be a link or a path to the file. I actually didn't know about it before now.

Reply:
I decided to specify the full path whenever it was relevant. This should be less confusing.